crates/conformance-test-utils/src/madara.rs

```diff
@@ -0,0 +1,92 @@
+use eyre::Result;
+use kakarot_rpc_core::client::constants::STARKNET_NATIVE_TOKEN;
+use starknet::core::types::FieldElement;
+use starknet::core::utils::get_storage_var_address;
+
+use crate::types::{ContractAddress, StorageKey, StorageValue};
+
+/// Generates the genesis storage tuple for pre-funding a Starknet address on Madara.
+///
+/// This function calculates the storage key for the balance of the ERC20 Fee Token
+/// contract using the provided Starknet address. The resulting tuple represents the initial
+/// storage of the Fee Token contract, where the account associated with the Starknet address is
+/// pre-funded with the specified `amount`.
+///
+/// # Arguments
+///
+/// * `starknet_address` - The Starknet address to be funded
+/// * `amount` - The amount of funds to allocate to the Starknet account.
+/// * `storage_offset_key` - The offset to be added to the storage key for the ERC20 Fee Token
+///
+/// # Returns
+///
+/// * `Result<((ContractAddress, StorageKey), StorageValue)>` - A tuple representing the initial
+///   storage of the Fee Token contract where the Starknet address has been pre-funded with the
+///   specified `amount`. Returns an error if the constant `STARKNET_NATIVE_TOKEN` could not be
```

Comment:
I think we decided to reduce the amount of comments we had. The function signature should be explicit enough to not need the `#Arguments` and `# Returns`.

Reply:
Ohh this is noted, I'll remove them to reduce some clutter 👌